### PR TITLE
Link the music page

### PR DESCRIPTION
**Pull request by BerePendzialek**
#8

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min'
```
Comment (macfrei) [inline .storybook/preview.js:2]: Doesn't this work: 
```suggestion
import { MemoryRouter } from 'react-router-dom'
```

File: src/components/MusicPage/MusicPage.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          >
+            <ShowFormatIntervalsT intervalsT={intervalsT} />
+          </IntervalsTSection>
+          <CooldownSection background={selectColor(cooldown.powerHigh)}>
+            <ShowFormatCooldown cooldown={cooldown} />
+          </CooldownSection>
+        </div>
+      </Wrapper>
+
+      <BackButton as={NavLink} to={{ pathname: '/' }}>
```
Comment (macfrei) [inline src/components/MusicPage/MusicPage.js:37]: I think you can write it like this, according to the react router dom documentation.
```suggestion
      <BackButton as={NavLink} path="/" >
```

File: src/components/MusicPage/MusicPage.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  text-align: justify;
+  padding: 2px;
+`
+
+const Wrapper = styled.section`
+  display: grid;
+  gap: 2px;
+`
+
+const WarmupSection = styled.div`
```
Comment (macfrei) [inline src/components/MusicPage/MusicPage.js:67]: If you name it `section` why not use a section element? Goes for `IntervalTSection` and `CooldownSection` as well.
```suggestion
const WarmupSection = styled.section`
```

File: src/components/WorkoutCard/WorkoutCard.js [hunk 4]
```diff
@@ -64,6 +65,11 @@ export default function WorkoutCard(props) {
           </CooldownSection>
         </div>
       </Wrapper>
+
+      <SelectButton as={NavLink} to={{ pathname: '/music', workout: workout }}>
```
Comment (macfrei) [inline src/components/WorkoutCard/WorkoutCard.js:69]: I am not sure how this routing works?

File: src/components/WorkoutCard/WorkoutCard.spec.js [hunk 5]
```diff
@@ -1,8 +1,9 @@
 import WorkoutCard from './WorkoutCard'
 import { render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
+import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min'
```
Comment (macfrei) [inline src/components/WorkoutCard/WorkoutCard.spec.js:4]: Should work like this.
```suggestion
import { MemoryRouter } from 'react-router-dom'
```

**Review by macfrei — COMMENTED** [review]
Looks good so far, I added a few comments you might want to consider! Keep it up 🙂

File: .storybook/preview.js (hunk 1 above)
Comment (BerePendzialek) [reply .storybook/preview.js:2]: Yes, it does. The link came automatic after I did the import

File: src/components/WorkoutCard/WorkoutCard.js (hunk 4 above)
Comment (BerePendzialek) [reply src/components/WorkoutCard/WorkoutCard.js:69]: As far as I understood, this is the object I am taking to the new location. I read this and got some help from the others. https://reactrouter.com/web/api/location

File: src/components/MusicPage/MusicPage.js (hunk 2 above)
Comment (BerePendzialek) [reply src/components/MusicPage/MusicPage.js:37]: I am currently using the ToLocation option from React Router and if I just write path= , it shows me this error: 


resolveToLocation(to, currentLocation),
  52 |   currentLocation
  53 | );
> 54 | const { pathname: path } = toLocation;
     | ^  55 | // Regex taken from: https://github.com/pillarjs/path-to-regexp/blob/master/index.js#L202
  56 | const escapedPath =
  57 |   path && path.replace(/([.+*?=^!:${}()[\]|/\\])/g, "\\$1");

I left it as it was.

File: src/components/MusicPage/MusicPage.js (hunk 3 above)
Comment (BerePendzialek) [reply src/components/MusicPage/MusicPage.js:67]: thanks! I´ll change it

File: src/components/WorkoutCard/WorkoutCard.js (hunk 4 above)
Comment (macfrei) [reply src/components/WorkoutCard/WorkoutCard.js:69]: Ah! Thanks :)

**Pull request merged**